tensorboard/plugins/image/tf_image_dashboard/tf-image-dashboard.html

```diff
@@ -44,6 +44,32 @@
             >Show actual image size</paper-checkbox>
           </div>
         </div>
+        <div class="sidebar-section">
+          <h3 class="tooltip-container">Brightness adjustment</h3>
+          <paper-slider
+            min="0"
+            max="2"
+            snaps
+            pin
+            step="0.01"
+            value="{{_brightnessAdjustment}}"
+            immediate-value="{{_brightnessAdjustment}}"
+            style="width: 100%;"
+          ></paper-slider>
+        </div>
+        <div class="sidebar-section">
+          <h3 class="tooltip-container">Contrast adjustment</h3>
+          <paper-slider
+            min="0"
+            max="500"
+            snaps
+            pin
+            step="1"
+            value="{{_contrastPercentage}}"
+            immediate-value="{{_contrastPercentage}}"
+            style="width: 100%;"
```

Comment:
Could we use embedded instead of inline styles? That decouples specifying the HTML from its design and makes it easier to make large-scale design changes.

Reply:
+1